.changeset/tasty-nails-fry.md

```diff
@@ -0,0 +1,14 @@
+---
+"thirdweb": minor
+---
+
+Adds getUsers function to query users on the server
+
+```ts
+import { getUsers } from "thirdweb/wallets";
+
+const users = await getUsers({
```

Comment:
still think its confusing to get an array here... i don't think i would expect that as a user and now i have to figure out which one in the array i care about

Reply:
Oh good catch I just forgot to update the changelog. It doesn't return an array anymore